Elizabeth_Theme/src/css/_colors.scss

```diff
@@ -0,0 +1,5 @@
+$lunar-resort-orange: #dfa356;
+$lunar-resort-blue: #415fa7;
+$lunar-resort-link-teal: #00ccFF;
+$theme-main-beige: #D6A261 ;
```

Comment:
Rather than making the variable so specific to a color, we should name the variable to it's usage instead.